doc/source/install.rst

```diff
@@ -192,32 +192,30 @@ installed), make sure you have `nose
 
 ::
 
-    $ nosetests pandas
-    ..........................................................................
-    .......................S..................................................
-    ..........................................................................
-    ..........................................................................
-    ..........................................................................
-    ..........................................................................
-    ..........................................................................
-    ..........................................................................
-    ..........................................................................
-    ..........................................................................
-    .................S........................................................
-    ....
+    $ import pandas as pd
+    $ pd.test()
+    Running unit tests for pandas
+    pandas version 0.18.0
+    numpy version 1.10.2
+    pandas is installed in pandas
+    Python version 2.7.11 |Continuum Analytics, Inc.| (default, Dec  6 2015, 18:57:58) [GCC 4.2.1 (Apple Inc. build 5577)]
```

Comment:
I would just manually wrap it, otherwise it will generate a horizontal scroll bar in the literal code block